langchain4j-anthropic/src/main/java/dev/langchain4j/model/anthropic/internal/api/AnthropicMessageContent.java

```diff
@@ -15,8 +16,27 @@
 public abstract class AnthropicMessageContent {
 
     public String type;
+    public CacheControl cacheControl;
 
     public AnthropicMessageContent(String type) {
         this.type = type;
     }
+
+    public AnthropicMessageContent(String type, CacheControl cacheControl) {
+        this.type = type;
+        this.cacheControl = cacheControl;
+    }
+
+    @EqualsAndHashCode
+    @JsonInclude(NON_NULL)
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    @JsonNaming(SnakeCaseStrategy.class)
+    @Getter
+    public static class CacheControl {
```

Comment:
please move it in its own file and register in `langchain4j-anthropic/src/main/resources/META-INF/native-image/dev.langchain4j/langchain4j-anthropic/reflect-config.json`